lib/generators/ruby_ui/install/install_generator.rb

```diff
@@ -56,7 +56,14 @@ def add_tailwind_css
 
       def add_tailwind_config
         say "Adding RubyUI config to tailwind config"
-        template "tailwind.config.js.erb", Rails.root.join("tailwind.config.js")
+
+        if File.exist?(Rails.root.join("tailwind.config.js")) # tailwindcss js package
+          template "tailwind.config.js.js-package.erb", Rails.root.join("tailwind.config.js")
+        elsif File.exist?(Rails.root.join("config/tailwind.config.js")) # tailwindcss-rails gem
+          template "tailwind.config.js.tailwindcss-rails.erb", Rails.root.join("config/tailwind.config.js")
+        else
```

Comment:
Rails apps using importmaps use tailwindcss-rails gem, the config file is located on `config` folder and uses a slightly different file content but I think it was enough to create 2 separated templates, I think it was easier than trying inject pieces of code conditionally.